packages/slate-editor/src/modules/editor-v4-rich-formatting/components/Text.tsx

```diff
@@ -26,13 +24,5 @@ export function Text({ attributes, children, leaf }: RenderLeafProps) {
         children = <u>{children}</u>;
     }
 
-    if (leaf[MarkType.SELECTION]) {
-        children = (
-            <span {...attributes} className={styled.selection}>
-                {children}
-            </span>
-        );
-    }
-
-    return <span {...attributes}>{children}</span>;
```

Comment:
`{...attributes}` is unnecessary here, as it is already rendered in `renderLeafExtensions`: https://github.com/prezly/slate/blob/824f64d237623935fb0927ad33055d3b463913c5/packages/slate-commons/src/lib/renderLeafExtensions.tsx#L22